### PR TITLE
Fix Unit test timezone bug [v8]

### PR DESCRIPTION
**Pull request by weresch**
## Description of the Change

Some unit tests recently added expected an EST timezone. This fix has it use the local environment timezone.

### CODE REVIEW

**Comment by pivotalgeorge** [comment]
passes on my machine in Pacific Time

File: command/v7/shared/app_summary_displayer_test.go [hunk 1]
```diff
@@ -702,13 +708,17 @@ var _ = Describe("app summary displayer", func() {
...
 							},
 						}
 					})
 
 					It("displays the message", func() {
-						Expect(testUI.Out).To(Say(`Rolling deployment currently CANCELING \(since Mon 29 Jul 13:32:29 EDT 2024\)`))
+						parsedTime, err := time.Parse(time.RFC3339, LastStatusChangeTimeString)
+						Expect(err).To(Not(HaveOccurred()))
+
+						timeString := parsedTime.Local().Format("Mon 02 Jan 15:04:05 MST 2006")
```
Comment (pivotalgeorge) [inline command/v7/shared/app_summary_displayer_test.go:720]: maybe we should extract the format string here for clarity

**Review by pivotalgeorge — APPROVED** [review]
passes on my machine in Pacific Time

File: command/v7/shared/app_summary_displayer_test.go (hunk 1 above)
Comment (weresch) [reply command/v7/shared/app_summary_displayer_test.go]: Good catch

**Comment by weresch** [comment]
Unit tests don't seem to be rerunning on the latest commit as the test failure messages are old. Closing this PR to open a fresh one.

**Pull request closed without merging**